js/hang-ui/src/Components/stats/style.css

```diff
@@ -0,0 +1,209 @@
+@import "./variables.css";
+
+.stats {
+	display: flex;
+	position: absolute;
+	inset: 0;
+	pointer-events: none;
+	z-index: 2;
+	width: 100%;
+	height: inherit;
+	container-type: inline-size;
+}
+
+.stats .stats__wrapper {
+	position: absolute;
+	width: 100%;
+	height: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	flex-direction: column;
+}
+
+.stats .stats__button {
+	position: absolute;
```

Comment:
I'm hesitant to make this is an overlay on the video. It looks really nice, but I wouldn't be surprised if users didn't like it blocking the video (even if just a tiny bit). 

What do you think about moving the button to the "control bar", like to the right of the fullscreen button? I think the panels are fine as an overlay though. I know we'd have to shrink the icon to make it look consistent with the other buttons, but I think putting it there makes more sense as that group (control bar buttons) represents ways to "interact" with the video, and a method to show stats about the video is a type of interaction (sort of).